core/core.lua

```diff
@@ -203,7 +204,10 @@ end
 local function GetColor(unit)
 	if unit then
 		local colorUnit = {}
-		if C.bar.marker and unit == "player" then
+		
+		if C.selfBar.enableCustomColor and UnitName("player") == UnitName(unit) then
```

Comment:
When player joins the raid the ```unit``` changes from "player" to "raidN"

Reply:
Good catch. It would make more sense to just use `UnitIsUnit(unit, "player")` though. (Needs to be cached with the other functions around line 36 as `local UnitIsUnit = _G.UnitIsUnit`

Reply:
Added this new function. Looks cleaner, tnx )